docs/ref/contrib/admin/actions.txt

```diff
@@ -34,6 +34,12 @@ models. For example, here's the user module from Django's built-in
     For more background on bulk deletion, see the documentation on :ref:`object
     deletion <topics-db-queries-delete>`.
 
+.. note::
+
+    .. versionadded:: 1.9
+
+    ``Change`` permission is needed for actions to accessible.
```

Comment:
I'd suggest adding a "How permissions interact with actions" section and explaining the rationale behind this a bit more. I imagine we may want to improve the API in the future to allow actions to be tied to permissions in some way (e.g. I can imagine an "Export items to CSV" action would be fine to keep if the user only has view permissions).